### DISCUSSION TITLE
Intermediate CA authoritie

### DISCUSSION THREAD

**Discussion by jtsfvieira** [iotaledger/firefly#8508] [Help & Support]
There is a way to work with firefly using a intermediate CA authorities? 

I'm trying to use the app in a corporate device and i'm getting this error message:

Fri, 10 May 2024 16:10:46 GMT
Generic: {"type":"error","payload":{"type":"client","error":"node error: error sending request for url (https://api.testnet.iotaledger.net/api/core/v2/info): error trying to connect: invalid peer certificate: UnknownIssuer"}}

Fri, 10 May 2024 16:10:36 GMT
Generic: {"type":"error","payload":{"type":"client","error":"node error: error sending request for url (https://api.testnet.iotaledger.net/api/core/v2/info): error trying to connect: invalid peer certificate: UnknownIssuer"}}

**Comment by begonaalvarezd** [comment]
Its seems like your machine is blocking the connection to the nodes, i dont know how your machine is configured but it looks like a problem in your machine, like your have a very strict firewall.

**Reply by jtsfvieira** [reply]
Its correct. In the other apps I need to include zscaler certificate in the app certificate store to work with https connections. 
Usually I found cacert.pem file or similar but in this case I no find any solution.
